app/models/dbedt_upload.rb

```diff
@@ -42,6 +52,22 @@ def make_active
     self.update! active: true
   end
 
+  def get_status(which)
+    if which == 'cats'
+      self.cats_status
+    else
```

Comment:
Yeah, I'm not a big fan of it either, but it seemed 'good enough' to me. What would you suggest? Have an elsif for series, and then either ignore anything else, or throw some kind of error?